src/extension.ts

```diff
@@ -262,6 +267,19 @@ function makeEndpointsProvider(): EndpointsProvider {
     return provider
 }
 
+function recordToolkitInitialization(activationStartedOn: number) {
+    try {
+        const activationFinishedOn = Date.now()
+        const duration = Math.abs(activationFinishedOn - activationStartedOn)
+
+        recordToolkitInit({
+            duration: duration,
+        })
+    } catch (err) {
+        getLogger().error(err)
```

Comment:
Is the logger guaranteed to be available? Otherwise this could throw a different error

Reply:
Not necessarily, I'll change that up